spring-kafka-avro/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot'
+	id 'org.springframework.aot.smoke-test'
+	id 'org.graalvm.buildtools.native'
+	id "com.github.davidmc24.gradle.plugin.avro" version "1.3.0"
+}
+
+apply plugin: "com.github.davidmc24.gradle.plugin.avro"
```

Comment:
The `id ...` in the plugins section should have applied the plugin already, if I'm not mistaken.